### PR TITLE
Poc detox

### PR DESCRIPTION
**Pull request by valpinkman**
copy of https://github.com/LedgerHQ/ledger-live-mobile/pull/2211

### CODE REVIEW

File: src/screens/Onboarding/steps/setupDevice/scenes/PairNew.tsx [hunk 1]
```diff
@@ -30,7 +30,12 @@ const Next = ({
   const { t } = useTranslation();
 
   return (
-    <Button type="main" size="large" onPress={onNext}>
+    <Button
+      type="main"
+      size="large"
+      onPress={onNext}
+      testID="Onboarding - Pair New Nano"
```
Comment (juan-cortes) [inline src/screens/Onboarding/steps/setupDevice/scenes/PairNew.tsx:37]: Won't this give you troubles? Ids shouldn't have spaces

File: e2e/models/onboarding/onboardingSteps.js [hunk 2]
```diff
@@ -0,0 +1,62 @@
+import * as bridge from "../../bridge/server";
+import * as testHelpers from "../../helpers";
+
+export default class OnboardingSteps {
+  static async waitForPageToBeVisible() {
+    await testHelpers.waitForElementByText("Get started");
+  }
+
+  static async startOnboarding() {
+    await testHelpers.tapByText("Get started");
```
Comment (juan-cortes) [inline e2e/models/onboarding/onboardingSteps.js:10]: Won't all these break if we change any of the wordings? Why not rely on ids like on other pages? 🤔

File: e2e/specs/password.spec.js [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+    await loadConfig("1AccountBTC1AccountETH", true);
+
+    await PortfolioPage.waitForPageToBeVisible();
+    await PortfolioPage.navigateToSettings();
+    await SettingsPage.navigateToGeneralSettings();
+    await GeneralSettingsPage.togglePassword();
+    await GeneralSettingsPage.enterNewPassword(CORRECT_PASSWORD);
+    await GeneralSettingsPage.enterNewPassword(CORRECT_PASSWORD); // confirm password step
+    await device.sendToHome(); // leave LLM app and go to phone's home screen
+    await delay(60001); // password takes 60 seconds of app inactivity to activate
```
Comment (juan-cortes) [inline e2e/specs/password.spec.js:25]: You could override https://github.com/LedgerHQ/ledger-live-mobile/blob/develop/src/constants.js#L21 if in a test run to not have to wait I guess.

**Review by juan-cortes — APPROVED** [review]
Few remarks, nothing blocking a merge for me

File: src/screens/Onboarding/steps/setupDevice/scenes/PairNew.tsx (hunk 1 above)
Comment (valpinkman) [reply src/screens/Onboarding/steps/setupDevice/scenes/PairNew.tsx:37]: Fixed

File: e2e/models/onboarding/onboardingSteps.js (hunk 2 above)
Comment (Justkant) [reply e2e/models/onboarding/onboardingSteps.js:10]: I would say that test-ids are useful but don't test in the way the user would, some more infos: https://testing-library.com/docs/queries/about/#priority

**Pull request merged**